### ISSUE TITLE
RDA elements related to dimensions (Chapter 6)

### ISSUE THREAD

**Issue by adams-liz** [rbms-bsc/DCRMR#154]
Hello!

Jessica and I are hoping to gather feedback related to the DCRM(G) incorporation. If you have any thoughts on the below issue, please share them by **January 3rd, 2023(!).** Thanks!

In new RDA, there are now Dimensions of still image and Dimensions of cartographic image elements. There are no similar elements for Extent of still image or Extent of cartographic image, and there are no other formats with dimension specific RDA elements. As a note, these elements are not on the soft deprecated list. Keeping in mind that there are several other manuals to incorporate and how this decision might impact that work, we have a couple options here:

1. Continue using Dimensions and not break out format by these two elements. We would then be very clear (similar to what is done with Details of illustrative content [https://bsc.rbms.info/DCRMR/phys-desc/Details-of-illustrative-content/](url)) in the area how the mapping works.
2. For the DCRM(G) incorporation, create a Dimensions of still image element, and eventually, a Dimensions of cartographic image one as well when we do that incorporation.

I lean toward the first option, because it seems like it will lead to less user confusion in the long run as we continue incorporating manuals (Why isn't there an Extent of still image element? Why aren't there other Dimensions of [format] elements? Where should I look generally?); it also lends itself to a tidier display. It is, however, technically not making full use of RDA elements. The obvious benefit of the second option is that it is more closely aligned with RDA. 

We welcome your thoughts and feedback!

Best wishes,

Liz

**Comment by io-uk** [comment]
I would go with the first option. Within the main Dimensions element, in the "Prerecording" section, under the condition "A manifestation carries a still image," there is the option to "Record the dimensions of the pictorial area of the image as a Manifestation: dimensions of still image" (with a link to that element) and in the same "Prerecording" section, under the condition "A manifestation carries a cartographic image," there is the option to "Record the dimensions of the face of the map, etc. as a Manifestation: dimensions of cartographic image" (with a link to that element). Thus, these elements are kind of covered in the main Dimensions element, even though the specific instructions are in a separate element. The specific elements also refer back to the main Dimensions element because they state "In addition, apply the basic instructions at Manifestation: dimensions as applicable" (in Dimensions of still image) and "Apply the basic instructions at Manifestation: dimensions" (in Dimensions of cartographic image). 
I don't catalogue graphics, so I don't really know about the specifics but looking at DCRM(G) it seems that what you measure depends on what kind of graphic material it is, i.e. with intaglio prints, you measure the plate mark, whereas for other material you measure the whole sheet. However, the specific instructions in the "Dimensions of still image" only refer to recording "the measurements of the pictorial area." If I interpret this correctly (I might not), then that would only cover the DCRM(G) case of intaglio prints but not the cases when we would measure the whole sheet; measuring the whole sheet would be covered by the instruction in the main Dimensions element for material with carrier type "Sheet". Therefore, we would have to go back and forth between the main element and the specific element anyway to cover everything, so I think it makes sense to just incorporate everything in the main element.
(I hope this makes sense; it is all a bit confusing as RDA just seems to complicate things unnecessarily).

Iris

**Comment by ladylazarus3** [comment]
Hi folks,

i agree with Iris and Liz that it makes sense to just incorporate everything in the main element.  Honestly, i think breaking it out will add additional inconsistency in tomorrow's legacy data making it that much harder to manage going forward and that the extra dimensions elements will unnecessarily complicate things.  There are already sooooo many obscure fields in our *current* systems that we have to beg and intercede with our library systems folks to display in our *current* discovery layers on preciously manicured and finitely allotted "screen real estate", and these "Dimensions of [Whathaveyous]" smack of just such a situation (and do we really need more?).  

Hypothetically, the dedicated dimensions elements might be useful if a super advanced and in the weeds researcher would want to do original research on measurements of intaglio prints using a dataset pulled from the catalog, but they are probably going to have to learn and use regular expressions to pull out and clean up that specific piece of data anyway.  (Admittedly, this is a pretty wild use case.)  i would be super interested to hear more about what graphics and cartographic (most especially in the case of maps) folks think, however.  

The happiest of holiday seasons to you all.  

kalan

**Comment by eblakedc** [comment]
If we do keep everything in the [dimensions](https://access.rdatoolkit.org/Content/Index?externalId=en-US_ala-cc95f0fd-c059-334d-b240-c7379e396c7d) element, we'll need to add a DCRMR elaboration of scope to show that instead of meaning just "A measurement of a carrier or a container of a manifestation" (the RDA definition) it can also include measurements of other things (e.g. plate mark of still image and pictorial area of still image, which are not the same thing).

Under [dimensions of still image](https://access.rdatoolkit.org/en-US_ala-03f0bb95-75c3-30ab-90e8-804bc3c36cdc), RDA  says "in addition, apply the basic instructions at Manifestation: [dimensions](https://access.rdatoolkit.org/Content/Index?externalId=en-US_ala-cc95f0fd-c059-334d-b240-c7379e396c7d) as applicable." In other words, the intention seems to be that "Dimensions" is used for dimensions of the carrier, and "Dimensions of still image" for the pictorial content that's on the carrier, with both being recorded. 

I can see the value of having the "Dimensions" element always refer to dimensions of the carrier, but it would have to be left blank for engravings (because dimensions of the plate mark are always given instead, since that's the part that doesn't vary from copy to copy). There isn't a "Dimensions of platemark" element, though, and creating separate elements for every not-carrier dimension that's conventionally recorded would be ridiculous.

Long story short: I agree that, for now, it's best to keep everything in the Dimensions element, and (for still images, at least) keep the DCRM(G) rule that you must always specify what has been measured.

Also, Happy New Year!

**Comment by jess-grz** [comment]
Based on the comments here and the consensus we've reached during our meetings, we will keep everything in Dimensions. We will note in the Dimensions element in DCRMR that instructions pertaining to still image resources map to [Dimensions of still image](https://access.rdatoolkit.org/en-US_ala-03f0bb95-75c3-30ab-90e8-804bc3c36cdc ) in the RDA Toolkit.
